### ISSUE TITLE
Problem with language

### ISSUE THREAD

**Issue by Methyx** [cesarwbr/react-input-emoji#56]
Hello !
Thanks for this wondefull component ! :-)
I have a problem with language
I have the last version :     "react-input-emoji": "^5.6.6",
When I try to select language with the Prop
language="fr"
I have this error in Chrome :
Uncaught Error: Dynamic require of "@emoji-mart/data/i18n/fr.json" is not supported

Did I miss something ?
Thanks in advance.

**Comment by cesarwbr** [comment]
Hi @Methyx!

Thank you for reporting the issue with the language prop. The error `Uncaught Error: Dynamic require of "@emoji-mart/data/i18n/fr.json" is not supported` has been fixed in version 5.6.8.

Please update your package to version 5.6.8 or higher by running:
```sh
npm install react-input-emoji@5.6.8
```
or if you're using Yarn:
```sh
yarn add react-input-emoji@5.6.8
```
After updating, the language prop should work as expected.

If you encounter any further issues, please let us know.

Thanks!

**Comment by Methyx** [comment]
Great ! Thanks a lot :-)
